inventory/consul_io.ini

```
@@ -52,3 +52,6 @@ kv_metadata=ansible/metadata
 availability = true
 available_suffix = _up
 unavailable_suffix = _down
+
+# if true then parse node name to ip
+node_to_ip = true
```

a **new** option is added here the suggested default may be `false`
```suggestion
node_to_ip = false
```